sdk/src/types/block/protocol/samples.rs

```diff
@@ -242,7 +242,7 @@ impl ProtocolParameters {
         self.rewards_parameters.final_target_rewards_rate = (self.token_supply()
             * self.rewards_parameters().reward_to_generation_ratio() as u64
             * self.mana_parameters().generation_rate() as u64)
-            >> self.mana_parameters().generation_rate_exponent() - self.slots_per_epoch_exponent();
+            >> (self.mana_parameters().generation_rate_exponent() - self.slots_per_epoch_exponent());
```

Comment:
We're 100% sure about that? Because it actually does change the computation

Reply:
https://doc.rust-lang.org/reference/expressions.html#expression-precedence `-` has precedence over `>>`

Reply:
Yeah I just read this page, interesting, I would have expected the opposite, thanks